content/community-targets/ATOM_Lite_ESP32.md

```diff
@@ -0,0 +1,61 @@
+# M5Stack ATOM Lite ESP32
+
+<img src="https://cdn.shopify.com/s/files/1/0056/7689/2250/products/1_84c35eb9-97c1-4537-ba67-8ecc18ffc82d_1200x1200.jpg?v=1598833664" width="400" height="400">
+
+[Product page](https://m5stack.com/products/atom-lite-esp32-development-kit)
+
+## Firmware
+
+Firmware can be flashed to the device with the following command:
+
+```
+nanoff --serialport <comx> --target ESP32_PICO  --update --baud 750000
+```
+
+A contributor was able to compile a custom nanoCLR image for this device via adding a new target to `cmake-variants.json` file by copying `Esp32_nanoCLR` target's settings and changing the `TARGET_SERIAL_BAUDRATE` value to `115200`.
```

Comment:
No need for this here. All the build configs are available on the respective folders.